examples/shouldi/shouldi/cargo_audit.py

```diff
@@ -0,0 +1,41 @@
+import json
+import asyncio
+from typing import Dict, Any
+
+from dffml.df.base import op
+from dffml.df.types import Definition
+
+package_src_dir = Definition(name="package_src_dir", primitive="str")
+cargo_audit_output = Definition(
+    name="golangci_lint_output", primitive="Dict[str, Any]"
+)
+
+
+class Cargo_AuditError(Exception):
```

Comment:
Let's not mix camel case and underscores

```suggestion
class CargoAuditError(Exception):
```